lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -45,26 +45,25 @@ def stub_email_alert_api_does_not_have_subscriber_subscriptions(id)
           .to_return(status: 404)
       end
 
-      def stub_email_alert_api_has_subscription(
-        id,
-        frequency,
-        title: "Some title",
-        subscriber_id: 1,
-        subscriber_list_id: 1000,
-        ended: false
-      )
+      def stub_email_alert_api_has_subscription(id, frequency, optional_overrides = {})
```

Comment:
@ChrisBAshton The method already allows you to override the values for title, subscriber_id, subscriber_list_id, and ended. Leaving them as named parameters makes it easier to find out what these values are.
It also means that you don't need a separate method to add these values.

Reply:
The list of keys was in two places (the `stub_email_alert_api_has_subscription` method parameters, and the `get_subscription_response` call below it), and in creating my `stub_email_alert_api_has_subscriptions` method I risked duplicating it a third time and the default values a second time (in my `get_subscription_response` call).

Felt like a good idea to extract that out so that we only have one place to update if we want to change a default value or add a new override. If we'd prefer the duplication I can do that.

Reply:
At this point (the first commit) you only have one method (`stub_email_alert_api_has_subscription`) so extracting out the default values it's necessary.

Reply:
Ah, agreed. Should I split it out so that it's repeated initially, then refactored into its own method in a subsequent commit?

Reply:
I think it's ok to keep the duplication. It's likely that the two test helpers will be used independently of each other, so I think it'll be more helpful to the person using them to be able to clearly see the defaults.